modules/op-security-with-custom-scc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='op-security-with-custom-scc_{context}']
+= Security with a custom SCC
+
+You can now leverage more security with custom SCC as it's easy to migrate pipelines that need to be run or built as root. However, if you would like to lock your installation further, you can only allow running the container as the user 1000 if the pod is asking for it and the default root is not allowed.
```

Comment:
```suggestion
You can improve security by using a custom <spell out what SCC stands for> (SCC) because it's easy to migrate pipelines that need to be run or built as root. To further lock down your installation, allow the container to run only as user 1000 when the pod asks for it and the default root is not allowed.
```